src/main.rs

```diff
@@ -707,7 +743,7 @@ fn get_setup(args: &[String]) -> Setup {
 
     let backend = audio_backend::find(backend_name).unwrap_or_else(|| {
         error!(
-            "Invalid `--{}` / `-{}`: {}",
+            "Invalid `--{}` / `-{}`: \"{}\"",
```

Comment:
Is there a way to replace all these identical strings with a constant?

Reply:
I will look into it. There certainly are a lot of places where the strings are the same.

Reply:
I just tried and it fails. A little Googling tells me it's because the 1st arg of the macros has to be a string literal. Best I could do would be to:

1. Write my own error macros (don't like that idea).
2. Use a closure or function (a little better IMHO).

Reply:
Shame, I agree not worth it.

Reply:
> Shame, I agree not worth it.

I'd love to be able to do `warn!(FOO, FOO_2, FOO_3);` It's just not currently in the cards.

Reply:
A macro doesn't look too bad actually. But i leave it up to you.